src/org/labkey/workflow/view/WorkflowViewBase.java

```diff
@@ -188,25 +190,28 @@ else if (variables.get(key) instanceof Date)
 
     }
 
-    public String navigationLinks(@Nullable String processDefinitionName, @NotNull String processDefinitionKey, @Nullable String processInstanceId)
+    public HtmlString navigationLinks(@Nullable String processDefinitionName, @NotNull String processDefinitionKey, @Nullable String processInstanceId)
     {
-        StringBuilder builder = new StringBuilder();
+        HtmlStringBuilder builder = HtmlStringBuilder.of();
+
         builder.append(PageFlowUtil.link("All workflows").href(new ActionURL(WorkflowController.BeginAction.class, getContainer())));
-        builder.append("\n&nbsp;&nbsp;\n");
+        builder.append(HtmlString.BR);
```

Comment:
Is this intentional to use HtmlString.BR instead of HtmlString.NBSP?

Reply:
Not intentional. Thanks.